src/server/rtu.rs

```diff
@@ -10,6 +10,7 @@ use crate::{
 };
 use futures::{select, Future, FutureExt as _};
 use futures_util::{SinkExt as _, StreamExt as _};
+use log::debug;
```

Comment:
I propose to use the `log::` prefix directly in the code instead of importing the module name. Log statements should be easily identifiable. They do not belong to the actual logic.

Reply:
on my projects I tend to remove the prefix for logs so I can swich easily from log to tracing, will fix